README.md

```diff
@@ -23,12 +23,20 @@ up-to-date. Think `docker build && kubectl apply` or `docker-compose up`.
 
 ## Install Tilt
 
-Installing the `tilt` binary is a one-step command:
+Installing the `tilt` binary is a one-step command.
+
+### macOS/Linux
 
 ```bash
 curl -fsSL https://raw.githubusercontent.com/windmilleng/tilt/master/scripts/install.sh | bash
```

Comment:
```suggestion
curl -fsSL https://raw.githubusercontent.com/tilt-dev/tilt/master/scripts/install.sh | bash
```

Reply:
it's on the other pr, just needed to rebase :wink: